src/main/pages/che-7/overview/proc_accessing-che-from-red-hat-applications-menu.adoc

```diff
@@ -0,0 +1,29 @@
+// accessing-che-from-openshift-developer-perspective
+
+[id="accessing-che-from-red-hat-applications-menu_{context}"]
+= Accessing Che from Red Hat Applications menu
+
+Starting with OpenShift Container Platform 4.2, a user can access Che workspace from the *Red Hat Applications* menu.
+
+.Prerequisites
+Che Operator is installed in OpenShift 4.2 or later.
+
+.Procedure
+
+. In the Openshift Container Platform, access the *Red Hat Applications* menu by clicking the button in the upper right corner of the main screen illustrated with a three by three cube.
++
+A drop-down menu with installed applications is displayed.
++
+image::installation/red-hat-application-menu-che-odp.png[link="{imagesdir}/installation/red-hat-application-menu-che-odp.png"]
+
+. Click the `Che` link. This creates a new workspace or redirects a user to an existing one. If your source link is a publicly accessible GitHub repository that contains a `devfile.yaml`, this file will be used to configure the workspace. Otherwise, a default workspace is created.
```

Comment:
```suggestion
. Click the *Che* link. This creates a new workspace or redirects to an existing one. For source links pointing to publicly accessible GitHub repositories that contain a `devfile.yaml` file, this file is be used to configure the workspace. Otherwise, a default workspace is created.
```